test/functional/notes_controller_test.rb

```diff
@@ -287,7 +287,8 @@ def teardown
         id: node.title.parameterize
 
     assert_response :success
-    assert_equal "First-time poster <a href='#{node.author.name}'>#{node.author.name}</a> submitted this #{time_ago_in_words(node.created_at)} ago and it has not yet been approved by a moderator. <a class='btn btn-default btn-sm' href='/moderate/publish/#{node.id}'>Approve</a> <a class='btn btn-default btn-sm' href='/moderate/spam/#{node.id}'>Spam</a>", flash[:warning]
```

Comment:
Keep this line and just replace the link with '/profile/#{@node.author.name}'

Reply:
@Gauravano, I have made the suggested changes, is it okay now?

Reply:
@Gauravano, updated.